video/out/opengl/ra.h

```diff
@@ -129,8 +138,8 @@ enum ra_buf_type {
 struct ra_buf_params {
     enum ra_buf_type type;
     size_t size;
-    // Creates a read-writable persistent mapping (ra_buf.data)
-    bool host_mapped;
+    bool host_mapped;  // create a read-writable persistent mapping (ra_buf.data)
+    bool host_mutable; // contents may be updated via buf_update()
```

Comment:
What's there to discard? Creating a buffer contains undefined data by default (unless you use initial_data).